splunk/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
- [ ] Let's capture the fact that this project was modernized from Ant to Maven including a subproject reorganization in the RELEASE_NOTES.md